indico/modules/events/static/tasks.py

```diff
@@ -49,6 +49,8 @@ def build_static_site(static_site):
 
         g.rh = rh
         ContextManager.set('currentRH', rh)
+
+        g.static_site = True
         ContextManager.set('offlineMode', True)
```

Comment:
maybe we could get rid of this one in favor of `g.static_site` (or rather `g.get('static_site')` since it's not set at all outside static site generation)